### PR TITLE
More about information

### PR DESCRIPTION
**Pull request by martinheidegger**
@spiegelmix: This PR contains an update on the about blurb with license information. Is this okay?

### CODE REVIEW

File: content/about_en.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
 
-School Lunch 2050 was established by the **FEAST Project** and is managed by *Maximilian Spiegelberg (FEAST Project)*. The future lunch sets were created by *Yuko Kobayashi & Yuko Matsuoka (FEAST Project)*, scenario illustrations by *Elie Tanabe (田辺エリ, Sustena)*, and the web design by *Martin Heidegger*. More information can be found at the [FEAST homepage] (feastproject.org/school-lunch-2050).
+School Lunch 2050 was established by the **FEAST Project**.
 
-The School Lunch 2050 homepage has been funded by the NIHU Interactive Communication Initiative.
+_Direction:_ Maximilian Spiegelberg ([FEAST Project][feast])<br/>
+_Future lunch sets:_ Yuko Kobayashi & Yuko Matsuoka ([FEAST Project][feast]) <br/>
+_Scenario illustrations:_ Elie Tanabe (田辺エリ, [Sustena Inc.][sustena])<br/>
+_Homepage Design & Code:_ [Martin Heidegger][mh]<br/>
+
```
Comment (spiegelmix) [inline content/about_en.md:12]: -Scenarios:_ FEAST team
_Translations:_ Yuko Kobayashi <br/>
_Research Assistance:_ Leo Porte & Haruka Tonegawa

File: content/about_en.md [hunk 2]
```diff
@@ -1,5 +1,36 @@
 **School Lunch 2050** is an educational and research project on climate change and food systems. It includes an installation and this homepage. At the moment it is focused on Japan, but we are happy to make more versions for other contexts.
```
Comment (spiegelmix) [inline content/about_en.md]: climate change, food systems, and the imagination of futures

File: content/about_en.md (hunk 1 above)
Comment (martinheidegger) [reply content/about_en.md:12]: I added these to the changes, Is the structure as you expect?

Also: Are there links for Leo & Haruka?

**Review by spiegelmix — COMMENTED** [review]
sorry, there are no links for Leo and Haruka. maybe scenarios can come first, then lunch sets, then translations...

File: content/about_en.md (hunk 2 above)
Comment (martinheidegger) [reply content/about_en.md]: Updated.

**Pull request merged**